extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -424,10 +426,14 @@ public function saveUserData() {
 			 */
 			$userIdentityBox = new UserIdentityBox( $user );
 
-			if ( !empty( $userData->website ) && 0 !== strpos( $userData->website, 'http' ) ) {
+			if ( !empty( $userData->website ) && strpos( $userData->website, 'http' ) !== 0 ) {
 				$userData->website = 'http://' . $userData->website;
 			}
 
+			if ( !empty( $userData->fbPage ) && strpos( $userData->website, self::FBPAGE_PREFIX ) !== 0 ) {
```

Comment:
That should be good :+1:  As long as we communicate the format of the link to the user :)